api/v1alpha1/openapi.yaml

```
@@ -3131,12 +3133,55 @@ paths:
+        "400":
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Status'
+        "401":
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Status' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)